api/sheets/get_variable.md

```diff
@@ -8,16 +8,27 @@ grand_parent: API
 
 # Get the value of a statistical variable at a given place and time.
 
-## `=DCGET(dcids, variable, date)`
+Given a list of [Place](https://datacommons.org/browser/Place) [DCIDs](/glossary.html), a [StatisticalVariable](https://datacommons.org/browser/StatisticalVariable), and optionally a date, get the measurements of the specified variable in the specified places with a date (if specified).
```

Comment:
I missed that Place and DCID had separate links when I was reading this in preview mode, maybe we can rephrase the sentence so it's clear there are tow links?

Reply:
Updated!